services/vod-to-live/service/.env.template

```diff
@@ -4,13 +4,6 @@ LOG_LEVEL=DEBUG
 
 PORT=11900
 
-# This variable is optional and is only needed for local development
```

Comment:
Not related to this file, but general comment:

Whenever I try to unpublish the playlist - v2l starts throwing `The specified blob does not exist.`error. Is it only happening for me?

Reply:
good point - the channel ID lacked the "channel-" prefix in that call but was included in the initial publish call.